app/src/main/java/io/github/droidkaigi/confsched2018/presentation/sessions/feedback/SessionsFeedbackFragment.kt

```diff
@@ -30,7 +35,7 @@ class SessionsFeedbackFragment : Fragment(), Injectable {
 
     @Inject lateinit var viewModelFactory: ViewModelProvider.Factory
 
-    private val onCurrentRankingChangeListener: FeedbackRankingView.OnCurrentRankingChangeListener
+    private val onCurrentRankingChangeListener
             = object : FeedbackRankingView.OnCurrentRankingChangeListener {
         override fun onCurrentRankingChange(view: FeedbackRankingView, currentRanking: Int) {
             val old = (sessionsFeedbackViewModel.sessionFeedback.value as? Result.Success)?.data!!
```

Comment:
I think `?: return` is more safely. What do you think?

Reply:
Right! I'll fix it